synapse/lib/cell.py

```diff
@@ -174,7 +178,7 @@ class Cell(s_eventbus.EventBus, s_telepath.Aware):
     '''
     cellapi = CellApi
 
-    confdefs = ()
+    confdefs: Tuple[Any, ...] = ()
```

Comment:
Awesome. I wasn't sure how to do this, I can clean up some of the cmdr typing warnings.